src/translations/maps/utils/index.js

```diff
@@ -0,0 +1,23 @@
+import { getDecreasedIndexes, getIncreasedIndexes } from './actionsOnIndexes';
+import { getListWithInsertedItems as visualStrategyInsert, getListWithRemovedItems as visualStrategyRemove } from './visuallyIndexed';
+import { getListWithInsertedItems as physicalStrategyInsert, getListWithRemovedItems as physicalStrategyRemove } from './physicallyIndexed';
+
+const alterUtilsFactory = (indexationStrategy) => {
+  if (indexationStrategy === 'visually') {
+    return {
+      getListWithInsertedItems: visualStrategyInsert,
+      getListWithRemovedItems: visualStrategyRemove
+    };
+  }
+
+  return {
+    getListWithInsertedItems: physicalStrategyInsert,
+    getListWithRemovedItems: physicalStrategyRemove
+  };
+};
```

Comment:
Having an object with defined all possible strategies may help easier maintain (add/remove) new strategies without changing the logic of the `alterUtilsFactory` function.
```suggestion
const alterStrategies = new Map([
  ['visually', { insert: visualStrategyInsert, remove: visualStrategyRemove}],
  ['physically', { insert: physicalStrategyInsert, remove: physicalStrategyRemove }],
]);

const alterUtilsFactory = (indexationStrategy) => {
  if (!alterStrategies.has(indexationStrategy)) {
    throw new Error(`Alter strategy '${indexationStrategy}' not exist`);
  }

  const { insert, remove } = alterStrategies.get(indexationStrategy);

  return {
    getListWithInsertedItems: insert,
    getListWithRemovedItems: remove,
  };
};
```